### PR TITLE
Addinf dichotomy direction to events

### PR DESCRIPTION
**Pull request by WalAmeni**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyDirection.java [hunk 1]
```diff
@@ -15,13 +15,13 @@ public enum DichotomyDirection {
     ES_PT("ES-PT"),
     PT_ES("PT-ES");
 
-    private final String direction;
+    private final String name;
 
     DichotomyDirection(String direction) {
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyDirection.java:20]: Could you change the name in the parameter as well ?

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java [hunk 2]
```diff
@@ -43,19 +43,19 @@ public DichotomyLogging(Logger businessLogger) {
     }
 
     public void logStartDichotomy(DichotomyDirection direction, Parameters parameters) {
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java:45]: "direction" argument is never used

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java [hunk 3]
```diff
@@ -43,19 +43,19 @@ public DichotomyLogging(Logger businessLogger) {
...
-        businessLogger.info("[{}] : Start dichotomy : minimum dichotomy index: {}, maximum dichotomy index: {}, dichotomy precision: {}", direction, parameters.getMinValue(), parameters.getMaxValue(), parameters.getPrecision());
+        businessLogger.info("Start dichotomy : minimum dichotomy index: {}, maximum dichotomy index: {}, dichotomy precision: {}", parameters.getMinValue(), parameters.getMaxValue(), parameters.getPrecision());
     }
 
     public void logEndOneDichotomy(DichotomyDirection direction) {
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java:49]: same, you dont need the argument

**Review by TheoPascoli — CHANGES_REQUESTED** [review]

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyDirection.java (hunk 1 above)
Comment (WalAmeni) [reply gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyDirection.java:20]: Ok

**Review by TheoPascoli — APPROVED** [review]

**Pull request merged**